### ISSUE TITLE
Uncaught TypeError: Cannot read property 'default' of undefined

### ISSUE THREAD

**Issue by paquitodev** [formvalidation/formvalidation#1369]
@nghuuphuoc  I've wrote my own validators (since my business logic requires them) and I'm must doing something wrong since I get this error all the time: 

> Uncaught TypeError: Cannot read property 'default' of undefined

This is the content of `validatorRIF.js` file, which is one of them:

    (function ($) {
        FormValidation.Validator.rif = {
            html5Attributes: {
                message: 'message',
                type: 'type'
            },
    
            /**
             * Return true if the input value is a valid RIF.
             *
             * @param {BootstrapValidator} validator The validator plugin instance
             * @param {jQuery} $field Field element
             * @param {Object} options Can consist of the following keys:
             * - message: The invalid message
             * - type: The type of RIF field. It can be V, E, J, P or G or the name of element which its value is the type
             * @returns {Boolean}
             */
            validate: function (validator, $field, options) {
                var value = $field;
    
                if (value === '') {
                    return true;
                }
    
                var $rif = value.val();
                var $rif_type = validator.getFieldElements(options.type).val().toUpperCase();
                var $last_number = $rif.substr($rif.length - 1);
                var $number = $rif.substr(0, 8);
    
                $valid = /^([VEJPG]{1})([0-9]{9}$)/.test($rif_type + $rif);
    
                if ($valid) {
                    if ($rif_type == "V") {
                        $sum = 1 * 4;
                    } else if ($rif_type == "E") {
                        $sum = 2 * 4;
                    } else if ($rif_type == "J") {
                        $sum = 3 * 4;
                    } else if ($rif_type == "P") {
                        $sum = 4 * 4;
                    } else if ($rif_type == "G") {
                        $sum = 5 * 4;
                    }
    
                    $n0 = $number.charAt(0) * 3;
                    $n1 = $number.charAt(1) * 2;
                    $n2 = $number.charAt(2) * 7;
                    $n3 = $number.charAt(3) * 6;
                    $n4 = $number.charAt(4) * 5;
                    $n5 = $number.charAt(5) * 4;
                    $n6 = $number.charAt(6) * 3;
                    $n7 = $number.charAt(7) * 2;
    
                    $sum += $n0 + $n1 + $n2 + $n3 + $n4 + $n5 + $n6 + $n7;
    
                    $mod = $sum % 11;
                    $last_val = 11 - $mod;
    
                    if ($last_val == 11 || $last_val == 10) {
                        $last_val = 0;
                    }
    
                    if ($last_number == $last_val) {
                        return true;
                    } else {
                        return false;
                    }
                }
    
                return false;
            }
        };
    }(jQuery));

The error is caused at line 553 of `formValidation.js` file, this line:

    case !!FormValidation.I18n[this.options.locale][validatorName]['default']:

I think the error is on the I18N then which leads me to the following question: how do I add my own translations for my own validators? If this is not the issue then, What is wrong? Why the error? I'm missing something in my validator?

**Comment by nghuuphuoc** [comment]
I will fix it (as #1343).
At the moment, you can define the default message for your validator as 

```javascript
// validatorRIF.js
(function($) {
    FormValidation.I18n = $.extend(true, FormValidation.I18n || {}, {
        'en_US': {
            rif: {
                'default': 'Please enter a valid RIF number',
            }
        }
    });

    FormValidation.Validator.rif = {
        ...
    }
}(jQuery));

```

**Comment by nghuuphuoc** [comment]
@paquitodev 
It's fixed in the latest code.
